README.MD

```diff
@@ -183,10 +183,16 @@ sudo tar -xzf bbb_nfs.tar.gz
 
 ## 2.4 Configuring host PC
 
+The purpose of 2.4 is to set up the necessary services to enable the devices to communicate with and
+access files on the host PC. The phases here correspond to [BBB booting](#22-bbb-booting) steps 2-5
+but are solely to enable the required services on host PC.
 Service wise there is different ways to do everything but the end goal is
```

Comment:
"Service wise there ARE different ways". Not your typo, but could be fixed aswell :)